test/performance/shmem_perf_suite/bw_common.h

```diff
@@ -271,10 +332,10 @@ void static print_atomic_results_header(perf_metrics_t metric_info) {
 
 void static print_results_header(perf_metrics_t metric_info) {
     printf("\nResults for %d PEs %lu trials with window size %lu "\
-            "max message size %lu with multiple of %lu increments\n",
-            metric_info.num_pes, metric_info.trials, metric_info.window_size,
-            metric_info.max_len, metric_info.size_inc);
-
+            "max message size %lu with multiple of %lu increments, "\
+            "\ntargeting %d remote PEs initiated from %d PEs\n", metric_info.num_pes,
+            metric_info.trials, metric_info.window_size, metric_info.max_len,
+            metric_info.size_inc, metric_info.sztarget, metric_info.szinitiator);
     printf("\nLength           %s           "\
             "Message Rate\n", metric_info.bw_type);
```

Comment:
Same here, trailing backslashes can be removed.  What you have is perfectly valid C, so this is not critical, but it may make the code a little easier to maintain.